admin/config/locales/en.yml

```diff
@@ -672,6 +672,7 @@ en:
           browser_title: Browser Title
           browser_title_note: The title appears in the top bar of a web browser and is shown in search engine results.
           browser_title_placeholder: "(defaults to content name)"
+          content_security_policy_note: This should be a single line string representing the content security policy for this content. This will be appended to the global security policy, if any.
```

Comment:
Can you add an example here for them to reference? Maybe we should move to a tooltip to allow more thorough explanation and like a copy pasta example.